### DISCUSSION TITLE
Installing on Debian 11 and Newest versions of php, apache...

### DISCUSSION THREAD

**Discussion by Chuck-K0XM** [magicbug/Cloudlog#1746] [General]
I have spent all weekend building a Debian 11 server and trying to install LAMP and cloudlog as shown on the web pages, without ANY luck.
Has anyone successfully deployed under Debian 11??
Chuck K0XM

**Comment by magicbug** [comment]
Hi Chuck,

Can't think there would be any issue, just a matter of getting things configured, did you end up with errors ?

Peter

**Reply by Chuck-K0XM** [reply]
kinda both....
I had to dig to find out how to install LAMP on Debian 11, and apache2 is operating fine, but php doesn't seem to be. And I cannot get to the cloudlog install point. The php version shows (7.4.03 I believe-not home right now) but the phptest.php attempt shows as not found- so have to figure where I screwed up.

**Reply by phl0** [reply]
Seems like PHP is only installed for command line and not for webserver?

**Reply by Chuck-K0XM** [reply]
Looks like I hosed the entire build....I have the php test working, but it is not shouwing by going to the URL (actually local IP) /phptest.php....

I got it at another location which is on me...I am going to rebuild the entire SSD tonite

**Reply by Chuck-K0XM** [reply]
well following the steps,there are a few items...so basically I am moving on. No more time to fight this. Git does not allow me to clone into /var/www directory- says it is not empty, and them the install script is no where to be found

**Reply by ChrisL79** [reply]
Had to install on ubuntu server 18xx as unable to install any version of php 7 for some reason on the 20x version. I deleted the www folder before cloning. Was the only way for me. Sudo rm -r www
You can specify install in any folder though.
This is a serious PITA to install. The instructions could do with a serious rewrite step by step and not aimed at the geek Linux user.
Once its all up and running its a cracking program, especially paired with the cloudlogoffline .apk.

**Reply by phl0** [reply]
The indtrictions are a wiki. Feel free to contribute and correct. As a geek Linux user it is often hard to think about how a non-pro would go through the steps. Plus verious different distros with its specific settings.

**Reply by ChrisL79** [reply]
Well there you go. Not everyone is a pro Linux user. Some people may have zero Linux experience. 
Write the wiki as you would for a total beginner coming from windows with no experience whatsoever.
The amount of people i know that have just given up on trying to install because of poor instructions and/or configs do not work out of the box.

**Reply by phl0** [reply]
You are right. But why not contribute where you fail to prevent others from doing the same.
This is not a professional software but open source. It lives from people contributing (also to documentation).

**Reply by ChrisL79** [reply]
I absolutely agree, but i am only a Linux beginner myself. I would have no idea if what i am saying is correct or not, or my way of doing it would introduce security flaws ect. 
I have had to install on a very old version of Ubuntu which in itself is not secure. That was the only way i could get a working instance. This is not what people should be doing, which is why it would be best coming from a Linux expert that is able to teach newcomers in a simple to understand step by step way and explain why each step is needed.
